hr_attendance_timesheet_adj/models/hr_attendance.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class HrAttendance(models.Model):
+    _inherit = 'hr.attendance'
+
+    worked_hours = fields.Char(
+        string='Worked Hours',
+        compute='_compute_worked_hours',
+        readonly=True,
+    )
```

Comment:
Doesn't widget="float_time" do what we want to achieve instead of overriding the field? In case it is necessary to override the field, I think `store=True` is missing.